query-engine/connector-test-kit-rs/query-tests-setup/src/datamodel_rendering/mod.rs

```diff
@@ -107,5 +99,4 @@ fn render_preview_features(provider: &str, excluded_features: &[&str]) -> String
 #[derive(Debug)]
 pub struct RenderedDatamodel {
```

Comment:
I left the type and didn't replace it with just `String` because the functions that accept this as argument have many parameters and having distinct types for them helps.